.github/workflows/prepare_deployment_branch.yaml

```diff
@@ -6,7 +6,7 @@ on:
     # At 23:00 on Monday and Wednesday (https://crontab.guru/#0_16_*_*_1,3)
```

Comment:
Please update comments related to the new time/day

Reply:
Updated!